internal/geometry/triangle.go

```diff
@@ -0,0 +1,76 @@
+package geometry
+
+import (
+	"fmt"
+	"math"
+)
+
+// TODO: In this file, create a new type called 'Triangle' with
+// fields 'A', 'B', and 'C' of type 'Point'.
+
+type Triangle struct {
+	A Point
+	B Point
+	C Point
+}
+
+// TODO: Add a method to the 'Triangle' type called 'Area' that returns the
+// area of the triangle.
+
+// Create function that adds X, Y values of A, B, C of Triangle, in order to be able to use it for the coming functions and calculations of Triangle values
+func (t Triangle) addAXY() float64 {
+	return t.A.X + t.A.Y
+}
+
+func (t Triangle) addBXY() float64 {
+	return t.B.X + t.B.Y
+}
+
+func (t Triangle) addCXY() float64 {
+	return t.C.X + t.C.Y
+}
```

Comment:
@nobe4 Okay like this, or should I try to simplify?
Feels a bit of a bit repetitive, but also can't currently come up with something more simple.

Reply:
Agreed, especially since you always use the 3 values together.

How about a function that returns the 3 sides. Something like:

```go
func (t Triangle) addXY() (float64, float64, float64) {
	return t.A.X + t.A.Y, t.B.X + t.B.Y, t.C.X + t.C.Y
}
```

Reply:
Actually for `Area()` I need a specific side each. I was thinking of doing 
`func (t Triangle) addXY() float64{
	return t.A.X + t.A.Y + t.B.X + t.B.Y + t.C.X + t.C.Y
}` 
instead before, as else I would have to do a loop or something to get to one `float64` value that can divide. Then I realized that I need the specific sides. 
I could do an array and use the respective array index for `Area()`. Do you think worth the effort or leave it like it is after all?

Reply:
look closer at the function I wrote, it's returning each side independently

Reply:
Added an improvement.
I do feel that declaring the sides as a variable once would be nicer, but maybe it's enough for now.